config/environments/development.rb

```diff
@@ -38,4 +38,15 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              Rails.application.secrets.smtp_address,
+    domain:               Rails.application.secrets.smtp_domain,
+    port:                 Rails.application.secrets.smtp_port,
+    user_name:            Rails.application.secrets.smtp_username,
+    password:             Rails.application.secrets.smtp_password,
+    authentication:       'plain',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.